driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -768,6 +810,42 @@ public ConfigBuilder withNotificationConfig(NotificationConfig notificationConfi
             return this;
         }
 
+        /**
+         * Sets a minimum severity for notifications produced by the server.
+         *
+         * @param minimumNotificationSeverity the minimum notification severity
+         * @return this builder
+         * @since 5.22.0
+         */
+        @Preview(name = "GQL-status object")
+        public ConfigBuilder withMinimumNotificationSeverity(NotificationSeverity minimumNotificationSeverity) {
+            if (minimumNotificationSeverity == null) {
+                notificationConfig = NotificationConfig.disableAllConfig();
+            } else {
+                notificationConfig = notificationConfig.enableMinimumSeverity(minimumNotificationSeverity);
+            }
+            return this;
+        }
+
+        /**
+         * Sets a set of disabled classifications for notifications produced by the server.
+         *
+         * @param disabledNotificationClassifications the set of disabled notification classifications
+         * @return this builder
+         * @since 5.22.0
+         */
+        @Preview(name = "GQL-status object")
+        public ConfigBuilder withDisabledNotificationClassifications(
+                Set<NotificationClassification> disabledNotificationClassifications) {
+            var disabledCategories = disabledNotificationClassifications == null
```

Comment:
I don't know if it's worth extracting the conversion of categories. It is the same code as in the `GqlNotificationConfig`.

Reply:
Closing as we have discussed this now.